### DISCUSSION TITLE
NutUI 所有版本区别 & 浏览器兼容程度

### DISCUSSION THREAD

**Discussion by richard1015** [jdf2e/nutui#663] [Q&A]
- @nutui/nutui   基于Vue2  视觉风格 JD APP 7.0规范  ，只能开发 h5
- @nutui/nutui-jdl   基于Vue2  视觉风格 JDL 规范  ，只能开发 h5
- @nutui/nutui@next   基于Vue3   视觉风格 JD APP 10.0规范 ，只能开发 h5
- @nutui/nutui-taro 基于Vue3   视觉风格 JD APP 10.0规范 ，必须基于 taro + vue3 框架 进行开发多端（多端指一套代码 部署多端环境  微信小程序 h5、等第三方小程序）
> @nutui/nutui@next 和 @nutui/nutui-taro 属于并行版本，存才部分的差异化，版本号始终保持一致， 支持在 iOS 9，Android 6.0以上操作系统

**Comment by yoyo837** [comment]
jdt又如何区别？文档看不到如何使用jdt和非jdt的用法。

**Reply by richard1015** [reply]
https://nutui.jd.com/jdt/
